_sass/components/_all.scss

```diff
@@ -2,6 +2,7 @@
 @import 'bar-chart-table';
 @import 'buttons';
 @import 'data-map';
+@import 'data-map-legend';
```

Comment:
fair point. I'll put it in the same file and separate code with comments instead.